draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -536,9 +550,12 @@ fragmentation transmitter. The following fields are common to all devices.
 SCHC F/R MUST concatenate FPort and LoRaWAN payload to retrieve the SCHC
 fragment as described in {{lorawan-schc-payload}}.
 
-* SCHC fragmentation reliability mode: ACK-Always.
+* SCHC fragmentation reliability mode:
+  * Unicast downlinks: ACK-Always.
+  * Multicast downlinks: No-ACK, reliability has be be ensured by the upper
+    layer. This feature is optional and MAY not be implemented by SCHC gateway.
 * RuleID: Recommended size is 8 bits in SCHC header.
-* Window index: encoded on W=1 bit, as per {{I-D.ietf-lpwan-ipv6-static-context-hc}}.
+* Window index (unicast only): encoded on W=1 bit, as per {{I-D.ietf-lpwan-ipv6-static-context-hc}}.
```

Comment:
I would keep W=1 for all downlinks (unicast + multicast)

Reply:
As far as i understand, there is no windows concept in No-ACK